test/validators/schema/AddressToRemoveSchema.test.ts

```diff
@@ -6,7 +6,7 @@ import { assertValidationErrors } from '../ValidationAssertions';
 
 const validator = new SchemaValidator(schema);
 
-describe('Applicant Details schema', () => {
+describe('AddressToRemove schema', () => {
```

Comment:
Good spot 🎖️ , seems like the convention we have been going for in these schemas would mean this would be:
```suggestion
describe('Address To Remove schema', () => {
```

Reply:
Yeah the naming conventions for these tests seem a bit all over the place - Controllers are in SentenceCase. I'll keep consistent within the schema tests though.